core-web/libs/sdk/angular/src/lib/models/dotcms.model.ts

```diff
@@ -301,7 +302,7 @@ interface DotCMSSiteContentType {
     permissionType: string;
 }
 
-interface DotCMSSiteParentPermissionable {
+export interface DotCMSSiteParentPermissionable {
     Inode: string;
     Identifier: string;
```

Comment:
camelCase

Reply:
I think this is a model from backend response